### ISSUE TITLE
Fix S3247 FP: Should not report on member access objects

### ISSUE THREAD

**Issue by pavel-mikula-sonarsource** [SonarSource/sonar-dotnet#9485]
Rule should not report on member-access symbols

```
        if (arg.Property is Fruit)                  // Noncompliant FP
        {
            _ = (Fruit)differentInstance.Property;  // Secondary
        }
```
